components/developer/jenkins/files/jenkins-standalone.xml

```diff
@@ -95,15 +101,36 @@
           </method_context>
         </exec_method>
 
+<!-- At the bare minimum, we want to restart and redeploy the WAR file and
+     its Jetty container when the service is refreshed (e.g. package update).
+     So we just kill it (if running) and hope SMF restart would do the rest.
```

Comment:
Of course... Sorry, but if it was only version bump, I'd merged it much faster :)